prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -156,7 +156,7 @@ presubmits:
     run_if_changed: "^(components/.*\\.yaml)|(test/presubmit-component-yaml.sh)|(sdk/python/.*)$"
     spec:
       containers:
-      - image: python:3.7
+      - image: python:3.9
```

Comment:
Would you mind pushing a commit to my branch here that has the changes you have in mind. You're obviously much more aware of the current testing setup and which things should change and which ones shouldn't. You should be able to push to my branch since I have allowed maintainers t commit to it.